pyzeebe/worker/worker.py

```diff
@@ -21,7 +22,8 @@ class ZeebeWorker(ZeebeTaskRouter):
     def __init__(self, name: str = None, request_timeout: int = 0, hostname: str = None, port: int = None,
                  credentials: BaseCredentials = None, secure_connection: bool = False,
                  before: List[TaskDecorator] = None, after: List[TaskDecorator] = None,
-                 max_connection_retries: int = 10, watcher_max_errors_factor: int = 3):
+                 max_connection_retries: int = 10, watcher_max_errors_factor: int = 3,
+                 max_task_count: int = 4):
```

Comment:
I think we should increase the default value to 32. That matches the default `maxJobsToActivate` value of Zeebe.